### ISSUE TITLE
About Batterymanager Kexts

### ISSUE THREAD

**Issue by mehmetyukcell** [teewhydope/HACKINTOSH_HP_440_G6_EFI_OPENCORE_BIG_SUR#1]
Dear @teewhydope , the folder include two kexts for battery manager. which I use the kexts acpibatterymanager and smcbatterymanager?

**Comment by teewhydope** [comment]
> Dear @teewhydope , the folder include two kexts for battery manager. which I use the kexts acpibatterymanager and smcbatterymanager?

i use both with no problem

**Comment by mehmetyukcell** [comment]
Thanks for your comments. But, I update the voodoops2(v2.7) and broken trackpad or touchpad gestures. Why?

**Comment by mehmetyukcell** [comment]
Can you share new EFI and Config for OEPNCORE 0.6.2

**Comment by teewhydope** [comment]
> Can you share new EFI and Config for OEPNCORE 0.6.2

updated. you can download now!

**Comment by teewhydope** [comment]
> Thanks for your comments. But, I update the voodoops2(v2.7) and broken trackpad or touchpad gestures. Why?

yeah. the latest voodoo broke touchpad for me too

**Comment by mehmetyukcell** [comment]
Yes, the final voodoops2 and voodooInput caused broke on touchpad and gestures for me too. Did you find a solution?

**Comment by teewhydope** [comment]
> Yes, the final voodoops2 and voodooInput caused broke on touchpad and gestures for me too. Did you find a solution?

nope. i reverted back to the previous kext and everything works fine again. the latest touchpad driver is broken

**Comment by mehmetyukcell** [comment]
https://osxlatitude.com/forums/topic/14175-hp-probook-450-g6-need-efi-for-oc-062/?tab=comments#comment-103745
